src/lib.rs

```diff
@@ -107,62 +129,58 @@ pub mod uuid {
         pub use uuidbuilder::*;
         pub use uuidutils::*;
     }
-    pub mod validator {
-        mod uuidvalidator;
-
-        pub use crate::types::ustatus::*;
-        pub use crate::types::validationresult::*;
-    }
-}
-
-pub mod transport {
-    pub mod datamodel {
-        mod uattributes;
-        mod ulistener;
-        mod umessagetype;
-        mod upayload;
-        mod upriority;
-        mod userializationhint;
-        mod utransport;
-
-        pub use uattributes::*;
-        pub use ulistener::*;
-        pub use umessagetype::*;
-        pub use upayload::*;
-        pub use upriority::*;
-        pub use userializationhint::*;
-        pub use utransport::*;
-
-        pub use crate::types::ustatus::*;
+    pub mod serializer {
+        mod longuuidserializer;
+        mod microuuidserializer;
+        mod uuidserializer;
+
+        pub use crate::types::serializationerror::*;
+        pub use longuuidserializer::*;
+        pub use microuuidserializer::*;
+        pub use uuidserializer::*;
     }
     pub mod validator {
-        mod uattributesvalidator;
+        mod uuidvalidator;
 
-        pub use uattributesvalidator::*;
+        pub use crate::types::validationerror::*;
+        pub use uuidvalidator::*;
     }
 }
 
-pub mod rpc {
-    mod calloptions;
-    mod rpcclient;
-    mod rpcmapper;
-    mod rpcresult;
-
-    pub use calloptions::*;
-    pub use rpcclient::*;
-    pub use rpcmapper::*;
-    pub use rpcresult::*;
+pub mod uprotocol {
```

Comment:
IMHO we are missing the module declarations for the additional proto files that you had added to the build.rs ...

Reply:
done